images/dotcms/Dockerfile

```diff
@@ -27,24 +28,37 @@ RUN chmod -R 666 /srv && find /srv/ -type d -exec chmod a+x {} \;
 
 # OpenJDK distributed under GPLv2+Oracle Classpath Exception license (http://openjdk.java.net/legal/gplv2+ce.html)
 # Alpine Linux packages distributed under various licenses including GPL-3.0+ (https://pkgs.alpinelinux.org/packages)
-FROM openjdk:8-jdk-alpine
+#ARG ARCH=arm64v8
+
+FROM alpine:3.11
 
 WORKDIR /srv
 
 RUN apk --no-cache upgrade \
-    && apk add --no-cache bash openssl ca-certificates gnupg grep sed tini nss s6-dns \
+    && apk add --no-cache bash openssl ca-certificates gnupg grep sed tini nss s6-dns openjdk8 go git gcc musl-dev libc6-compat tomcat-native curl \
     && update-ca-certificates 
 
-RUN apk --no-cache add libc6-compat tomcat-native curl && apk add libsass --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/
+RUN apk add libsass --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/
+
+
+
+# Update java security preferences for ARM
+#RUN sed -i 's/NativePRNGBlocking:SUN/NativePRNGNonBlocking:SUN/g'  /usr/lib/jvm/default-jvm/jre/lib/security/java.security
+#RUN sed -i 's/random/urandom/g'  /usr/lib/jvm/default-jvm/jre/lib/security/java.security
 
-# This is a nasty hack for libjsass, Alpine 3.9 no longer has a deprecated symlink in /lib
-RUN [ ! -e /lib/ld-linux-x86-64.so.2 ] &&  ln -s $(readlink /lib64/ld-linux-x86-64.so.2) /lib/ld-linux-x86-64.so.2 || :
 
-# Dockerize distributed under MIT license (https://github.com/jwilder/dockerize)
-ENV DOCKERIZE_VERSION v0.6.0
```

Comment:
The big difference is that we have to build `dockerize` because it it not available as a package on ARM